Sources/Fuzzilli/Modules/Statistics.swift

```diff
@@ -75,9 +78,23 @@ public class Statistics: Module {
             self.ownData.totalExecs += 1
             self.currentExecs += 1
         }
-        fuzzer.registerEventListener(for: fuzzer.events.InterestingProgramFound) { _ in
+        fuzzer.registerEventListener(for: fuzzer.events.InterestingProgramFound) { ev in
             self.ownData.interestingSamples += 1
             self.ownData.coverage = fuzzer.evaluator.currentScore
+
+            if ev.program.typeCollectionStatus == .success {
+                self.ownData.interestingSamplesWithTypes += 1
+            }
+
+            if ev.program.typeCollectionStatus != .notAttempted {
```

Comment:
I think these three ifs should only happen if `ev.isImported == false` since otherwise you'll count them multiple times. Is that correct?

Reply:
hmm I guess it depends when we actually ran the collection https://github.com/googleprojectzero/fuzzilli/blob/master/Sources/Fuzzilli/Fuzzer.swift#L371
so maybe we don't avoid sending flag in event?

Reply:
Ah yeah, you're right. I guess once the runtime type collection is stable we can see if we still want these detailed statistics, and if not, then we can remove the new flag in the event again. So let's just go with the simplest solution for now (i.e. yours)